grafana-formula/grafana-formula.changes

```diff
@@ -1,3 +1,9 @@
+-------------------------------------------------------------------
+Fri Jan 22 08:31:16 UTC 2021 - Witek Bedyk <witold.bedyk@suse.com>
+
+- Version 0.4.0
+- Add SLES 15 SP3 and openSUSE 15.3 to supported versions
```

Comment:
Hehe, sorry to be so picky, but can you then write `openSUSE Leap 15.3`? :-)

Other than that, this looks good.

Reply:
done